app/views/viewall/students.html.erb

```diff
@@ -1,6 +1,39 @@
 <h2 align="center">Complete Student Member Status</h2>
 <hr>
 
+<style type="text/css">
+.label {
```

Comment:
@aniarya82

Why is this necessary? I think a screenshot will better explain this, if it has any yo improvements.